### ISSUE TITLE
Datepicker validation

### ISSUE THREAD

**Issue by johnsvenn** [BRACKETS-by-TRIAD/admin-generator#3]
Hi

Firstly, I just want to say what a fantastic piece of work you've made available to the community.

I'm just playing around with Brackets, so I'm not sure if this a configuration issue due to a mistake on my part or an actual bug.

It appears that the form validation for the date field isn't working:

In a migration I've added a field as so:

`$table->date('published_date')->nullable();`

An then used the admin generator:

In a form when I pick a date e.g. 12.10.2017

I see the following error: `The published_date must be in the format YYYY-MM-DD kk:mm:ss.`

The actual generated html looks like this:

```
<div class="col-sm-8">
  <div class="input-group input-group--custom">
    <div class="input-group-addon"><i class="fa fa-calendar"></i></div>
    <input type="hidden" id="published_date" name="published_date" placeholder="Select date" data-input="" class="form-control input flatpickr form-control-danger" data-vv-id="_d9sz6fynr" aria-required="false" aria-invalid="true" value="2017-10-12 00:00:00">
    <input class="form-control input flatpickr flatpickr-input" placeholder="Select date" type="text" readonly="readonly">
  </div>
  <div class="form-control-feedback form-text">The published_date must be in the format YYYY-MM-DD kk:mm:ss.</div>
</div>
```

So despite the error message, the field does contain a valid date.

This was built with 1.0.3.

**Comment by palypster** [comment]
Hi @johnsvenn, thank for your feedback. We're happy that you like it :)

About your issue, yes, this is a bug and we actually have this on our list already. You can of course fix that for yourself in your already generated code. We're definitely going to fix that in `admin-generator` very soon.

**Comment by johnsvenn** [comment]
Thanks, I'll do so.

**Comment by palypster** [comment]
Fixed in v1.0.4 with #4
